tensorflow/core/kernels/data/dataset_test_base.cc

```diff
@@ -202,7 +202,7 @@ Status DatasetOpsTestBase::InitThreadPool(int thread_num) {
         "The `thread_num` argument should be positive but got: ", thread_num);
   }
   thread_pool_ = absl::make_unique<thread::ThreadPool>(
-      Env::Default(), ThreadOptions(), "inter_op", thread_num);
+      Env::Default(), ThreadOptions(), "intra_op_thread_pool", thread_num);
```

Comment:
call this simply "test_thread_pool"

Reply:
Rename it to `test_thread_pool` in this commit https://github.com/tensorflow/tensorflow/pull/27648/commits/9cc83b27b1833f721c69dc877ef285000070162c